go-fuzz-build/main.go

```diff
@@ -665,6 +665,20 @@ func (c *Context) clonePackage(p *packages.Package) {
 		c.copyFile(f, dst)
 	}
 
+	// copy subdirs which aren't packages
+	dir := filepath.Dir(p.GoFiles[0])
```

Comment:
Is there a guarantee here that p.GoFiles is non-empty?

Reply:
Go-package contains at least one go-file, so GoFiles is not empty anyway.